vivisect/__init__.py

```diff
@@ -49,6 +50,10 @@
 
 import vivisect.analysis.generic.emucode as v_emucode
 
+def guid():
```

Comment:
this is also in visgraph.graphcore.  is there a better central location for guid to be placed?